### DISCUSSION TITLE
My account and my Gist don't appear on Google Search

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21859] [New to GitHub]
<p>I searched my username on Google with site:GitHub.com and I got no results</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/darkwarrior703">@darkwarrior703</a>,</p>
<p>Thanks for being here, I’m afraid we have no control over what Google indexes and caches, so you would need to consult them for further information. You may be able to find some starting info here:</p>
<p><a href="https://support.google.com/webmasters/answer/34397" rel="nofollow noopener">https://support.google.com/webmasters/answer/34397</a></p>
<p>Of course, if you have any questions specific to <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> itself, please feel free to let us know!</p>

**Reply by dariomangoni** [reply]
The problem is that, to trigger the indexing of a page, we need to have writing access to the page itself and we, as simple users, we do not.
Here the description: [URL Inspection Tool](https://support.google.com/webmasters/answer/9012289?hl=en#request_indexing)

So I think you may trigger a domain search to Google, not us!

**Reply by Tannerpierce225** [reply]
Google.com

**Comment by Mousetrapp1212** [comment]
https://www.google.com/?safe=active&ssui=on

**Comment by elijaugh444-eng** [comment]
https://www.google.com

**Comment by anamul8877haque-rgb** [comment]
Help guide
